packages/flutter/test/widgets/restoration_mixin_test.dart

```diff
@@ -127,6 +127,7 @@ void main() {
 
     // Rename the existing bucket.
     state.injectId('newnewnew');
+    await tester.pump();
```

Comment:
Why the extra pump?

Reply:
Removed. It is not necessary. Must be left over from debugging something.